.kitchen.yml

```diff
@@ -2,6 +2,9 @@
 verifier:
   name: inspec
 
+transport:
+  name: ssh
+
```

Comment:
Why is this needed?

Reply:
The rsync transport doesn’t support Inspec, apparently.

Reply:
I ran a `kitchen test` without specifying the transport, and I didn't see any issues. What problems were you running into?

Reply:
Just kidding-- Seems problems only appear on `blue`, which I forgot to run it on the first time around.